feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -92,10 +93,21 @@ func configureDeviceId(t *testing.T, dut *ondatra.DUTDevice) {
 	gnmi.Replace(t, dut, gnmi.OC().Component(p4rtNode).Config(), &c)
 }
 
+// configureInterfaceType configures interface type on the DUT.
+func configureInterfaceType(i *oc.Interface) *oc.Interface {
+	i.Type = if_type
+	return i
+}
+
 // configurePortId configures p4rt port-id on the DUT.
 func configurePortId(t *testing.T, dut *ondatra.DUTDevice) {
+	d := gnmi.OC()
 	portName := dut.Port(t, "port1").Name()
-	gnmi.Replace(t, dut, gnmi.OC().Interface(portName).Id().Config(), portId)
+	curr_intf := &oc.Interface{Name: ygot.String(portName)}
```

Comment:
Inline the `Type` value to `ethernetCsmacd` here without adding the additional functions.

Reply:
I have updated the code.